release-1.8/release_notes_draft.md

```diff
@@ -64,6 +64,12 @@ fundamental aspect of a secure cluster.
   have deployed Heapster in aggregated API server mode, ensure that you
   upgrade Heapster as well.
 
+* Advanced auditing has graduated from `v1alpha1` to `v1beta1` with some
+  changes to the default behavior. The webhook and log file now output the
+  `v1beta1` event format, the log file defaults to JSON encoding, and the
+  `--audit-policy-file` requires `kind` and `apiVersion` fields specifying
+  what format version the `Policy` is using.
+
```

Comment:
Advanced auditing has graduated from `v1alpha1` to `v1beta1` with the following changes to the default behavior. 
* Webhook and the log file now output the `v1beta1` event format
* The log file defaults to JSON encoding
* The  `--audit-policy-file` requires `kind` and `apiVersion` fields specifying what format and version the `Policy` is using.

Reply:
done, thanks